hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -212,6 +212,19 @@ def _bank_add_signals(
     }
 
 
+@bp.route("/bank/<bank_name>/content/<content_id>", methods=["DELETE"])
+def bank_delete_content(bank_name: str, content_id: int):
+    """
+    Remove a signal from a bank.
+    """
+    storage = persistence.get_storage()
+    bank = storage.get_bank(bank_name)
+    if not bank:
+        abort(404, f"bank '{bank_name}' not found")
+
+    storage.bank_remove_content(bank.name, content_id)
+    return {"message": "Done"}
```

Comment:
i took this from another endpoint but happy to make this more specific

Reply:
I think this is fine, I can't remember why we added it.